UPGRADING.md

```diff
@@ -24,6 +24,9 @@ Swift:
 
 + BSGConfigurationErrorDomain
 + BSGConfigurationErrorCode
++ config.persistUser
++ [config persistUserData]
++ [config deletePersistedUserData]
```

Comment:
What is the end user intent for accessing these? If there isn't one, then they shouldn't be public on config.

Reply:
Fair point.  Removed and appropriate @interfaces added to tests.